packages/core-database-postgres/lib/connection.js

```diff
@@ -203,36 +203,19 @@ module.exports = class PostgresConnection extends ConnectionInterface {
    * @param  {Number} height
    * @return {Array}
    */
-  async buildDelegates (maxDelegates, height) {
+  buildDelegates (maxDelegates, height) {
     if (height > 1 && height % maxDelegates !== 1) {
       throw new Error('Trying to build delegates outside of round change')
     }
 
-    let data = await this.db.rounds.delegates()
-
-    // NOTE: At the launch of the blockchain we may not have enough delegates.
-    // In order to have enough forging delegates we complete the list in a
-    // deterministic way (alphabetical order of publicKey).
-    if (data.length < maxDelegates) {
-      const chosen = data.map(delegate => delegate.publicKey)
-
-      const fillerWallets = chosen.length
-        ? await this.db.rounds.placeholdersWithout(maxDelegates - data.length, chosen)
-        : await this.db.rounds.placeholders(maxDelegates - data.length)
-
-      data = data.concat(fillerWallets)
-    }
-
-    // logger.info(`got ${data.length} voted delegates`)
     const round = Math.floor((height - 1) / maxDelegates) + 1
```

Comment:
Shouldn't we be worried about `Math.floor(110 / 1.1)` being `99`?
Maybe something like: `bignum(height).minus(1).dividedBy(maxDelegates).plus(1).toNumber()`? (yuck!)